apps/console/src/features/applications/components/forms/general-details-form.tsx

```diff
@@ -25,6 +25,7 @@ import { useSelector } from "react-redux";
 import { Divider, Icon } from "semantic-ui-react";
 import { AppState, UIConfigInterface } from "../../../core";
 import { ApplicationManagementConstants } from "../../constants";
+import {applicationConfig} from "../../../../extensions";
```

Comment:
There should be a lint error here.
Install the [relevant eslint plugin ](https://github.com/wso2/identity-apps/blob/master/docs/SET_UP_DEV_ENVIRONMENT.md#eslint-ide-plugin) and validate.